### ISSUE TITLE
Dark Theme Regression: Lots of low-contrast text (OS/Firefox specific)

### ISSUE THREAD

**Issue by jaller94** [vector-im/element-web#23241]
### Steps to reproduce

**This regression seems to come from my Firefox or operating system as overnight all versions of Element Web are affected. Chromium is not affected. I don't remember upgrading anything yesterday.**

1. Where are you starting? What can you see?

![Screenshot 2022-09-08 at 13-25-39 Element Matrix Games Night](https://user-images.githubusercontent.com/10872136/189110744-d438b49f-283d-4b21-a544-8891ff136339.png)

When using dark mode, lots of text does not change to a light font color.

#### What did you expect?
High contrast of text and its background.

### Operating system

Manjaro Linux

### Browser information

Mozilla Firefox 104.0b9

### URL for webapp

staging.element.io, develop.element.io and call.element.io

### Application version

1.11.4 and all others I've tested

### Will you send logs?

No

**Comment by t3chguy** [comment]
Does this happen if you try a fresh/clean Firefox profile?

**Comment by KoffeinKaio** [comment]
Happened to me too.
Creating a new profile did help (so did switching browsers or launching firefox in private mode).
Strangely the same problem now happens on github itself

**Comment by KoffeinKaio** [comment]
traced it back to an addon "Facebook Container". As soon as you enable it, text goes dark

**Comment by jaller94** [comment]
Yes, this seems to be a regression of Facebook Container 2.3.2 to 2.3.3.

https://github.com/mozilla/contain-facebook/issues/858
